docs/form-controls/textarea.md

```diff
@@ -1,21 +1,21 @@
 ﻿---
 title: Text Area
-summary: Textarea allows users to input multiple lines of text into a form.
+summary: Text area allows users to input multiple lines of text into a form.
```

Comment:
This is both a named thing in Pelican and it’s also the HTML name of the thing. Let’s leave is as Textarea

Reply:
Then why is the title of this page "Text Area"?